mock-caas/src/main/java/com/sequenceiq/caas/grpc/service/MockUserManagementService.java

```diff
@@ -506,6 +507,34 @@ public void listRoles(UserManagementProto.ListRolesRequest request, StreamObserv
         responseObserver.onCompleted();
     }
 
+    @Override
+    public void getActorWorkloadCredentials(com.cloudera.thunderhead.service.usermanagement.UserManagementProto.GetActorWorkloadCredentialsRequest request,
+                                            io.grpc.stub.StreamObserver<com.cloudera.thunderhead.service.usermanagement
+                                                .UserManagementProto.GetActorWorkloadCredentialsResponse> responseObserver) {
+
+        final int keyType17 = 17;
+        final int keyType18 = 18;
+        final int saltType = 4;
+        GetActorWorkloadCredentialsResponse.Builder respBuilder = GetActorWorkloadCredentialsResponse.getDefaultInstance().toBuilder();
+        respBuilder.addKerberosKeysBuilder(0)
+            .setSaltType(saltType)
+            .setKeyType(keyType17)
+            .setKeyValue("testKeyValue17")
+            .setSaltValue("NonIodizedGrainOfSalt")
```

Comment:
does this mock need to actually provide values that will work when pushed to a real IPA? if so, these clearly aren't going to work. You can use these values:
```
 'keys': [{'key': {'enctype': 18,
                   'keyvalue': b'\xbb?=i\xd8\x98\x89\x15\x98K#g\xc4\x98\xc8W'
                               b'\xfe\x9d\xb5T\xcck\xfeP\x89\t,\rYl\xe8\xc8'},
           'salt': {'salt': b'd1QmHyuRC8xW0jbf', 'type': 4}},
          {'key': {'enctype': 17,
                   'keyvalue': b"\x19\xdf\x8e\xc4W\x8fz0\xd9'\x13_"
                               b'\x9f\x93\t\xd1'},
           'salt': {'salt': b'VgUdOYXP4iuw1DRw', 'type': 4}}],
```

Reply:
and hashedpassword: {SSHA512}e1NTSEE1MTJ9ODk1SGNYSFFzZWIzS2dtTEdPYTZZRDcvSXJEM29Od0pqdGVlWjlhMmZvdTJsaTE5T240ejhFWDVyTTNiY0xCOHorRVY4Y05RckhyMApKbisrR1RXaG5YcFZZME5xVUcxYVRWRkpaelJ4UjFOMVZXaE9XRkJXVEdSTWIyTlZVVGR2Cg==

Reply:
This is not going to be in real IPA, this is for unit tests and local uses. For real real IPA, it must me real ums.